webapp/src/ts/services/enketo.service.ts

```diff
@@ -630,11 +636,14 @@ export class EnketoService {
 
         $('form.or').trigger('beforesave');
 
-        if (docId) {
-          return this.update(docId);
-        }
-        return this.create(formInternalId);
-      })
+        return this.ngZone.runOutsideAngular(() => this._save(formInternalId, form, geoHandle, docId));
+      });
+  }
+
+  private _save(formInternalId, form, geoHandle, docId?) {
```

Comment:
Do we have another `save()` ? if not then we can remove the underscore, it's already private.

Reply:
We do have another save. One that calls this :)

Reply:
Oh couldn't see it before, thanks!